SCClassLibrary/Common/Audio/SynthDef.sc

```diff
@@ -565,8 +569,14 @@ SynthDef {
 	// methods for special optimizations
 
 	// only send to servers
-	send { arg server, completionMsg;
-		var servers = (server ?? { Server.allRunningServers }).asArray;
+	send { | server = \allRunningServers, completionMsg |
+		var servers;
+		if (server.isNil, {
+			"Provided server is nil, SynthDef may not be sent".warn;
+		});
+		servers = if ((server == \allRunningServers)  || server.isNil,
```

Comment:
generally, it is more efficient to write:

` if ((server == \allRunningServers)  or: { server.isNil },`